Update margin-trim content (#23026)
mdn/content: files/en-us/web/css/margin-trim/index.md

- `none`
  - : Margins are not trimmed by the container.

- `block`
  - : Container truncate the margins provided to the block children where they adjoin the container's edge, without affecting the margins provided to the container.

- `block-start`
  - : Margins of the first block child with container's edge will be trimmed to zero.

- `block-end`

>   - : Margins of last block child with the container's edge will be truncated to zero.

Reviewer: ```suggestion
  - : Margin of last block child with the container's edge is trimmed to zero.
```

- `inline`

>   - : Container truncate the margins provided to the inline children where they adjoin the container's edge, without affecting spacing at the beginning and end of the       row.

Reviewer: ```suggestion
  - : Margins provided to the inline children where they adjoin the container's edges are trimmed to zero without affecting the spacing at the beginning and end of the row.
```

- `inline-start`

>   - : Margins between the container's edge and the first inline child will be truncated to zero.

Reviewer: ```suggestion
  - : Margin between the container's edge and the first inline child is trimmed to zero.
```

- `inline-end`

>   - : Margin between the container's edge and the last inline child will be truncated to zero.

Reviewer: ```suggestion
  - : Margin between the container's edge and the last inline child is trimmed to zero.
```

```css
span:last-child {
  margin-right: 0;
  margin-left: 0;
}
```

It is a pain having to write another rule to achieve this, and it is also not very flexible. Instead, `margin-trim` could fix it:

```css
article {
  margin-trim: inline-end;
  /* … */
}
```

> Similarly, to remove left margins with the container's edge:

Reviewer: ```suggestion
Similarly, to remove left margin with the container's edge:
```
